packages/components/src/components/form/Range/Range.tsx

```diff
@@ -157,12 +168,7 @@ export const Range = ({
 
     return (
         <div className={className}>
-            <Input
-                {...props}
-                type="range"
-                disabled={disabled}
-                trackStyle={disabled ? disabledTrackStyle : trackStyle}
-            />
+            <Input {...props} type="range" disabled={disabled} trackStyle={trackStyle} />
             {labels?.length && (
                 <LabelsWrapper count={labels.length} $width={labelsElWidth}>
```

Comment:
How about adding `margin-top: ${spacingsPx.xs};` to `LabelsWrapper` so it 's not that close to the range component itself?

Reply:
3737afd8cd